src/Assessment/Assess.cs

```diff
@@ -204,10 +210,10 @@ public bool BeginAssessment()
                             SqlServicesVM.Add(discoverySiteMachine.MachineId);
                     }
 
-                    if (discoverySiteMachine.MachineId.Contains("vmwaresites"))
+                    if (discoverySiteMachine.MachineId.Contains("vmwaresites") || (UserInputObj.AzureMigrateSourceAppliances.Contains("import") && discoverySiteMachine.MachineId.Contains("importsites")))
                         AzureVMWareSolution.Add(assessmentSiteMachine);
 
-                    if (addMachineToGeneralVM)
+                    if (!UserInputObj.AzureMigrateSourceAppliances.Contains("import") && addMachineToGeneralVM)

```

Comment:
This is wrong,
We should only have "addMachineToGeneralVM" if it is not added to VMWARE, or any other source, it needs to be added to general VM,

make the addMachineToGeneralVm = false, in case it is import or gets added to vmware machines.

Should not augment this condition.

Reply:
without setting addMachineToGeneralVM as false, all the cases are working fine because of this condition:
!UserInputObj.AzureMigrateSourceAppliances.Contains("import")